android/src/main/java/io/mosip/tuvali/verifier/GattService.kt

```diff
@@ -23,8 +24,8 @@ class GattService {
       BluetoothGattService.SERVICE_TYPE_PRIMARY
     )
 
-    val identityChar = BluetoothGattCharacteristic(
-      IDENTITY_CHARACTERISTIC_UUID,
+    val identifyRequestChar = BluetoothGattCharacteristic(
+      IDENTIFY_REQUEST_CHAR_UUID,
       BluetoothGattCharacteristic.PROPERTY_WRITE_NO_RESPONSE or BluetoothGattCharacteristic.PROPERTY_WRITE,
```

Comment:
@PuBHARGAVI Are both types of write properties required here?

Reply:
yes @krishnakumar4a4. on ios side we are writing to this characteristic with response` connectedPeripheral.writeValue(messageData, for: characteristic, type: .withResponse)` and android side we are writing to it without response `writeChar?.writeType = BluetoothGattCharacteristic.WRITE_TYPE_NO_RESPONSE`